### PR TITLE
M3-5829: Add contextual help to linode create page

### PR DESCRIPTION
**Pull request by codyfinn**
## Description

Adds contextual help to the Linode create flow. In particular it adds a link to the _Getting Started_ and _Choosing a Compute Instance Type and Plan_ docs. For the time being I altered the `DocsLink` component to have a prop called `onClick` that allows the developer to specify what should happen when a user clicks the link in addition to the base functionality of the link. Another alteration I made was to the `TabbedPanel` component. I added the `docsLink` prop to allow the developer to add a `DocsLink` on the same paper as the panel across from the header.

## How to test

1. Navigate to the Linode create page
2. Ensure at the top right of the page there is a `DocsLink` labeled _Getting Started_ and that it links to the _Getting Started_ doc
3. Ensure that a GA event is sent that has the content of `{ category: 'Linode Create Flow', action: 'Click:link', label: 'Getting Started' }`
4. Navigate back to the Linode create page
5. Ensure that to the right of the header for the plan selection section is a `DocsLink` that links to the _Choosing a Compute Instance Type and Plan_ docs
6. Ensure that a GA event is sent that has the content of `{ category: 'Linode Create Flow', action: 'Click:link', label: 'Choosing a Plan' }`

### CODE REVIEW

**Review by bnussman — COMMENTED** [review]
Google Analytics events looked good, but I did find one small UI issue. On a very narrow viewport, the docs link is very close to the title.

Screen Shot 2022-09-13 at 4 40 39 PM

**Review by bnussman — APPROVED** [review]
Looks good! 🚀 

On an unrelated note, I noticed that these docs links all open in a new window and not in a new tab in Safari. Chrome opens in new tab.

File: packages/manager/src/components/TabbedPanel/TabbedPanel.tsx [hunk 1]
```diff
@@ -73,6 +78,9 @@ export const TabbedPanel: React.FC<CombinedProps> = (props) => {
     innerClass,
     tabs,
     handleTabChange,
+    docsLink,
+    onDocsLinkClick,
+    docsLinkLabel,
```
Comment (DevDW) [inline packages/manager/src/components/TabbedPanel/TabbedPanel.tsx:83]: `docsLinkLabel` currently isn't used anywhere:

Screen Shot 2022-09-14 at 1 33 02 PM

On second thought, are `onDocsLinkClick` and `docsLinkLabel` needed at all? Seems like the label and click handler get passed well enough via `docsLink`

File: packages/manager/src/features/linodes/LinodesCreate/SelectPlanPanel.tsx [hunk 2]
```diff
@@ -553,6 +556,15 @@ export const SelectPlanPanel: React.FC<CombinedProps> = (props) => {
...
       initTab={initialTab >= 0 ? initialTab : 0}
+      docsLink={docsLink}
+      docsLinkLabel="Choosing a Plan"
+      onDocsLinkClick={() => {
+        sendEvent({
+          category: 'Linode Create Flow',
+          action: 'Click:link',
+          label: 'Choosing a Plan',
+        });
+      }}
```
Comment (DevDW) [inline packages/manager/src/features/linodes/LinodesCreate/SelectPlanPanel.tsx:567]: tied to my other comment -- could `docsLinkLabel` and `onDocsLinkClick` be eliminated?

File: packages/manager/src/components/TabbedPanel/TabbedPanel.tsx [hunk 3]
```diff
@@ -82,11 +90,24 @@ export const TabbedPanel: React.FC<CombinedProps> = (props) => {
...
-        )}
+        <Grid className={classes.header}>
+          {header !== '' && (
+            <Grid item xs={6}>
+              <Typography variant="h2" data-qa-tp-title>
+                {header}
+              </Typography>
+            </Grid>
+          )}
+          {docsLink && onDocsLinkClick && (
```
Comment (DevDW) [inline packages/manager/src/components/TabbedPanel/TabbedPanel.tsx:101]: we can give this the ternary treatment instead (like `docsLink ? ( JSX...) : null`)

**Review by DevDW — COMMENTED** [review]
The functionality looks good, I'm wondering if a couple of the new props can be eliminated though as it seems they're redundant

**Review by DevDW — APPROVED** [review]
LGTM 🚢

**Pull request merged**